opt/se-logger/liveupdate.py

```diff
@@ -206,7 +208,7 @@ def get_messages(self, byteiterator):
         elif state == 8:
           # Reading message checksum.
           if len(data) == 20 + length + 2:
-            data = "".join(data)
+            data = bytes(data)
```

Comment:
```suggestion
            data = byterray(data)
```
This is so that SEDecryptor.decript will not complain about its data being immutable.

Reply:
Got it! Thx!